platforms/android/demo/src/main/java/com/mapzen/tangram/android/MainActivity.java

```diff
@@ -88,6 +189,63 @@ public void onLowMemory() {
         view.onLowMemory();
     }
 
+    @Override public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
+        final ArrayList<SceneUpdate> sceneUpdates = new ArrayList<>(1);
+        final String apiKey = "vector-tiles-tyHL4AY";
```

Comment:
remove key and replace with value that is read from gradle.properties?

1. add to `~/.gradle/gradle.properties`:
```
tangramDemoApiKey=the_api_key
```

2. add to demo app `build.gradle`:
```
def DEMO_API_KEY = '"' + tangramDemoApiKey + '"';

buildTypes {
    debug {
      debuggable true
      jniDebuggable true
      buildConfigField 'String', 'API_KEY', DEMO_API_KEY
    }
  }
```

3. update this line to: `final String apiKey = BuildConfig.API_KEY;`